tests/network-tests/src/flows/membership/updatingVerifyAccount.ts

```diff
@@ -0,0 +1,44 @@
+import { FlowProps } from '../../Flow'
+import {
+  BuyMembershipHappyCaseFixture,
+  MemberProfileData,
+  UpdateProfileHappyCaseFixture,
+} from '../../fixtures/membership'
+
+import { extendDebug } from '../../Debugger'
+import { FixtureRunner } from '../../Fixture'
+import { generateParamsFromAccountId } from '../../fixtures/membership/utils'
+import { MembershipMetadata } from '@joystream/metadata-protobuf'
+
+export default async function updatingVerifyAccount({ api, query }: FlowProps): Promise<void> {
```

Comment:
This is not the actually testing the validator profile verification. To test this I think you would need to create a fixture like `RemarkModeratePostsFixture`. But that would be a lot of effort. @zeeshanakram3 in your opinion is this test worth the effort or can we skip it here ?